src/addRemove.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-use-before-define, import/prefer-default-export */
+
+let toDoArr = [];
+const addInput = document.querySelector('.insert-text');
+const form = document.getElementById('form');
+let addValue = '';
+
+const changeValue = (indexChange, value) => {
+  toDoArr[indexChange - 1].description = value;
+  localStorage.setItem('toDo', JSON.stringify(toDoArr));
+};
+
+const add = () => {
+  toDoArr.push({ description: addValue, completed: false, index: toDoArr.length + 1 });
+  localStorage.setItem('toDo', JSON.stringify(toDoArr));
+};
+
+const remove = (element) => {
+  toDoArr = toDoArr.filter((toDo) => toDo.index !== parseInt(element.parentNode.id, 10));
+
+  toDoArr.forEach((toDo, idx) => {
+    toDoArr[idx].index = idx + 1;
+  });
+
+  element.parentNode.parentNode.remove();
+
+  localStorage.setItem('toDo', JSON.stringify(toDoArr));
+  displayList();
+};
+
+export const displayList = () => {
```

Comment:
- [ ] **[OPTIONAL]** For better modularization, this file should be all about adding and removing. I would recommend you move this `displayList` function to a different file.